imports/plugins/included/product-variant/client/templates/products/productSettings/productSettings.js

```diff
@@ -67,15 +67,85 @@ Template.productSettings.helpers({
   }
 });
 
-Template.productSettingsGridItem.helpers({
-  displayPrice: function () {
+// Template.productSettingsGridItem.helpers({
```

Comment:
Let's not have commented out code. That's what source control is for. Let's remove it or leave it.